views/lexicon/tables/form-tables.html.twig

```diff
@@ -123,7 +123,7 @@
                         <td>my_label</td>
                         <td>my_value</td>
                         <td class="centered">
-                            {{ html.edit_button({ 'placement': 'left' }) }}
+                            {{ html.edit_button({ 'placement': 'left', 'class': 'u-margin-right-small' }) }}
```

Comment:
Will still pass this but it feels like a button group may be more appropriate here to manage spacing?